### ISSUE TITLE
After Party

### ISSUE THREAD

**Issue by iancrowther** [tancredi/nodeconf-london-2016#44]
Create page with after party details

**Comment by tancredi** [comment]
Do we have content for this?

**Comment by iancrowther** [comment]
yep, we have the garden room until 19:00, and then we can use the two bars/restaurants on site

after that, its open season..

**Comment by iancrowther** [comment]
http://www.barbican.org.uk/restaurants-bars/bonfire

or

http://www.barbican.org.uk/restaurants-bars/osteria

**Comment by iancrowther** [comment]
do you need any more information to begin this?

**Comment by iancrowther** [comment]
what do you think? stil want?

**Comment by orlacoding** [comment]
Is this still required? The schedule already has 'Drinks reception' at the end.

**Comment by iancrowther** [comment]
This may go in this weekend, but if not, will close
